feature/staticroute/static_route_support/README.md

```diff
@@ -0,0 +1,114 @@
+# RT-1.26: Basic static route support
+
+## Summary
+
+- Static route ECMP must be supported
+- Static route metric must be supported
+- Static route Administrative Distance / Preference must be supported
+- ```set-tag``` attribute must be supported for static routes
+- Disabling recursive nexthop resolution must be supported
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+#### Initial Setup:
+
+*   Connect DUT port-1, port-2 and port-3 to ATE port-1, port-2 and port-3 respectively
+*   Configure IPv4/IPv6 addresses on DUT and ATE the interfaces
+*   Configure one IPv4 destination i.e. ipv4-network connected to ATE
+*   Configure one IPv6 destination i.e. ipv6-network connected to ATE
+
+### RT-1.26.1 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route ECMP
+
+*   Configure IPv4 static routes:
+    *   Configure one IPv4 static route i.e. ipv4-route-a on the DUT for destination ipv4-network with the next hop of port-1
+    *   Configure another IPv4 static route i.e. ipv4-route-b on the DUT for destination ipv4-network with the next hop of port-2
+*   Configure IPv6 static routes:
+    *   Configure one IPv6 static route i.e. ipv6-route-a on the DUT for destination ipv6-network with the next hop of port-1
+    *   Configure another IPv6 static route i.e. ipv6-route-b on the DUT for destination ipv6-network with the next hop of port-2
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on both port-1 and port-2 and ECMP works
+
+### RT-1.26.2 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route metric
+
+*   Configure metric of ipv4-route-b and ipv6-route-b to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/metric
+*   Validate that the metric is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/metric
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on port-1 only
```

Comment:
It seems the routes from RT-1.26.1 would be used since they are default metric and therefore traffic is sent to both port1 and port2?   Looks like the new ipv4/6-route-b is not used?  Please define some route and metric that forces the DUT to make a routing decision that results in only one port being used.

Reply:
```
ipv[4|6]-route-a has next-hop = ATE port-1 IP
ipv[4|6]-route-b has next-hop = ATE port-2 IP
```

RT-1.26.1 ensures ECMP works [line 34](https://github.com/openconfig/featureprofiles/blob/46788e97668c975a98719db28f9d7bed57febfe7/feature/staticroute/static_route_support/README.md?plain=1#L34). Now that we increase metric of ipv[4|6]-route-b routes (which have next-hop of port-2) traffic will be received only on port-1 of ATE (port-2 should not get any traffic). I will clarify this